### PR TITLE
Feature/create entity anonymization configurations

### PR DESCRIPTION
**Pull request by GalloisLuca**

### CODE REVIEW

File: src/Resources/mappings/ShopUser.yaml [hunk 1]
```diff
@@ -1,4 +1,35 @@
...
+        salt:
+            faker: text
+            args: [60]
+        password:
+            faker: sha256
+        lastLogin:
+            faker: dateTime
+        emailVerificationToken:
+            faker: uuid
+        passwordResetToken:
```
Comment (oallain) [inline src/Resources/mappings/ShopUser.yaml:18]: should be null.

File: src/Resources/mappings/ShopUser.yaml [hunk 2]
```diff
@@ -1,4 +1,35 @@
...
+            faker: uuid
+        passwordRequestedAt:
```
Comment (oallain) [inline src/Resources/mappings/ShopUser.yaml:20]: Should be null.

File: src/Resources/mappings/ShopUser.yaml [hunk 3]
```diff
@@ -1,4 +1,35 @@
...
+            faker: dateTime
+        verifiedAt:
+            faker: dateTime
+        locked:
+            faker: boolean
+            args: [100]
+        credentialsExpireAt:
+            faker: dateTime
```
Comment (oallain) [inline src/Resources/mappings/ShopUser.yaml:28]: Should be null.

File: src/Resources/mappings/Order.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+        billingAddress: ~
+        payments: ~
+        currencyCode:
+            faker: currencyCode
+        localeCode:
+            faker: locale
+        customerIp:
+            faker: ipv6
+        notes:
+            faker: text
```
Comment (oallain) [inline src/Resources/mappings/Order.yaml:13]: Should be null.

File: src/Resources/mappings/Payments.yaml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+Sylius\Component\Core\Model\Payment:
+    properties:
+        details:
+            faker: text
```
Comment (oallain) [inline src/Resources/mappings/Payments.yaml:4]: Should be null.

**Review by oallain — CHANGES_REQUESTED** [review]

File: src/Resources/mappings/ShopUser.yaml (hunk 1 above)
Comment (GalloisLuca) [reply src/Resources/mappings/ShopUser.yaml:18]: i can't put a null value for now, i will implement it in an other PR

**Review by oallain — APPROVED** [review]

**Pull request merged**